build-tools/packages/build-tools/src/type-test-generator/typeTestUtils.ts

```diff
@@ -0,0 +1,289 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { readJsonSync } from "fs-extra";
+import path from "node:path";
+import { IExtractorConfigPrepareOptions } from "@microsoft/api-extractor";
+import { existsSync } from "node:fs";
+import { Project, SourceFile } from "ts-morph";
+import { BrokenCompatTypes } from "../common/fluidRepo";
+import { buildTestCase, TestCaseTypeData } from "../typeValidator/testGeneration";
+import { getFullTypeName, getNodeTypeData, TypeData } from "../typeValidator/typeData";
+import { PackageJson } from "../common/npmPackage";
+
+/**
+ * Checks the package object to verify that the specified dependency exists
+ * @param packageObject - package.json object
```

Comment:
Nit: As a general rule, we should use full `Sentance case.` in comments, including param comments.